### PR TITLE
Tillstep3

### PR DESCRIPTION
**Pull request by vikas-bisht**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -15,7 +15,11 @@ import { TodoService } from './todo.service';
     AppComponent, ListComponent,EditComponent,AddComponent
   ],
   imports: [
-    BrowserModule,FormsModule
+    BrowserModule,FormsModule,RouterModule.forRoot([
+      {path:'',redirectTo:'todo',pathMatch:'full'},
```
Comment (Rahul-Khera-Codes) [inline src/app/app.module.ts]: is todo route  defined  {path:'',redirectTo:'todo',pathMatch:'full'},?

Comment (vikas-bisht) [reply src/app/app.module.ts]: no sir! now solved it

File: src/app/app.module.ts [hunk 2]
```diff
@@ -16,7 +16,6 @@ import { TodoService } from './todo.service';
   ],
   imports: [
     BrowserModule,FormsModule,RouterModule.forRoot([
-      {path:'',redirectTo:'todo',pathMatch:'full'},
```
Comment (Rahul-Khera-Codes) [inline src/app/app.module.ts]: is your app working without path '' ??
fix this

Comment (vikas-bisht) [reply src/app/app.module.ts]: yes sir its working!

**Pull request merged**